plugins/linux-v4l2/v4l2-output.c

```diff
@@ -93,7 +93,9 @@ static bool try_connect(void *data, int device)
 	vcam->frame_size = width * height * 2;
 
 	char new_device[16];
-	sprintf(new_device, "/dev/video%d", device);
+	if (device < 0 || device > MAX_DEVICES)
```

Comment:
Wouldn't this be
```c
device >= MAX_DEVICES
```

Reply:
Right, thanks for catching this. The code has been updated and the commit squashed.